src/main/java/emissary/server/EmissaryServer.java

```diff
@@ -122,7 +122,12 @@ public ServerCommand getServerCommand() {
     public Server startServer() {
         // do what StartJetty and then JettyServer did to start
         try {
-            // Resource.setDefaultUseCaches(false);
+
+            if (this.cmd.shouldStrictMode()) {
+                LOG.info("Strict mode activated");
+                System.setProperty("strict.mode", "true");
```

Comment:
This goes back to a debate we had when first starting this, whether we should include this as a command line option or a line in a config file.  I think the conclusion we came to was that a command line argument would be easier to manage in a start script/jenkins build?  In that case, I'm not sure how to get that command in EmissaryServer and pass it to EmissaryNode without messing with the constructor.

Reply:
Look at HttpCommand, we add properties like:
`setSystemProperty(EmissaryNode.NODE_SCHEME_PROPERTY, scheme);`
Then in EmissaryNode, the constructor creates the var:
`this.nodeScheme = System.getProperty(NODE_SCHEME_PROPERTY, "http");`
Startup requires an EmissaryNode in the constructors...so in the method you edited in Startup, you should be able to call `this.node.getStartupMode()` instead of `System.getProperty("strict.mode")`

Still uses the command line, no config file needed, and follows the same pattern as the other options.

Reply:
👆 This is the way

Reply:
Ok Mandalorian 😄

Reply:
Ended up running myself in circles due to improper debugger usage, but I think I worked out what you meant.